### ISSUE TITLE
The replicas value  doesn't auto refresh in the "xxx_Scaling.json" file after editing scaling range

### ISSUE THREAD

**Issue by v-xinda** [microsoft/vscode-azurecontainerapps#330]
**Environment:** VSCode.dev
**Container Apps Build Version:** v0.4.2-alpha.0
**Resource Group Build Version:** v0.6.3-alpha.0

**Repro Steps:**
1. Create a container app and expand it.
2. Right click "Scaling" node -> Edit Scaling Range… -> Set the range -> Enter.
3. Right click "Scaling" node -> View Properties.
4. Check whether the replicas value auto refreshes in the `xxx_Scaling.json `file.

**Expect:**
The replicas value auto refreshes in the `xxx_Scaling.json `file.

**Actual:**
The replicas value  doesn't auto refresh in the `xxx_Scaling.json `file.
![image](https://user-images.githubusercontent.com/57662131/229453989-fb7d471c-4feb-4c68-9b1a-62ae5c36c385.png)

**More Info:**
1. This issue also reproduces after executing "Add Scale Rule..." action.
2. This issue also reproduces after executing "Switch Ingress Visibility.../Edit Target Port...".
3. Reload VSCode can fix this issue.

**Comment by nturinski** [comment]
I don't think this is `.dev` specific. Probably is due to the data being fetched on the container app itself. A refresh of the Azure view will fix this issue as well.

We need to change how "refresh" works for most children node since most of them are not updating the resource node anymore.
